recipes/pyppeteer/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyppeteer" %}
+{% set version = "0.0.25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51fe769b722a1718043b74d12c20420f29e0dd9eeea2b66652b7f93a9ad465dd
+  patches:
+    - setup.patch
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pyppeteer-install = pyppeteer.command:install
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
The lib seems to be compatible with py35

Reply:
The py35 support is stated to experimental and needs one extra dependence: py-backwards, that is not on conda-forge. Hence patch.

Reply:
We don't ship py35 on conda-forge, so this discussion is moot.

Reply:
That being said, we do need the patch so it builds.

Reply:
actually, maybe we don't need the patch. We should try it without.

Reply:
Conda will not try and build python 3.5, so removing the patch is safe.